src/presets/deno/preset.ts

```diff
@@ -1,12 +1,7 @@
 import { builtinModules } from "node:module";
-import MagicString from "magic-string";
-import { findStaticImports } from "mlly";
 import { defineNitroPreset } from "nitropack/kit";
 import { writeFile } from "nitropack/kit";
-import { isAbsolute, resolve } from "pathe";
-
-// nitro/src/rollup/plugin/import-meta.ts
-const ImportMetaRe = /import\.meta|globalThis._importMeta_/;
+import { resolve } from "pathe";
 
 const denoDeploy = defineNitroPreset(
```

Comment:
We should probably create new preset and enable via compatibilityDate (I will push)

Reply:
90a6484577da7f6ca75204d8ce8de1a19244e241

Reply:
Ah looks like you already pushed that 👍